src/index.css

```diff
@@ -2,6 +2,7 @@
 @import url("https://vivirfashion.netlify.app/style.css");
 @import url("https://vivirfashion.netlify.app/products.css");
 @import url("https://vivirfashion.netlify.app/cart/cart.css");
+@import url("https://vivirfashion.netlify.app/wishlist/wishlist.css");
```

Comment:
Just a suggestion pls put all your css files in one base file in your component library and then import that file only.